src/Components/Home.js

```diff
@@ -0,0 +1,49 @@
+import React, { useEffect } from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+import { NavLink } from 'react-router-dom';
+import { fetchCurrencies } from '../Redux/Currencies/currencies';
+import styles from './Styles/Home.module.css';
+import Coin from './Coin';
+import Search from './Search';
+/* eslint-disable */
+const Home = ({ set }) => {
+  const data = useSelector((state) => state.coins);
+  const dispatch = useDispatch();
+  useEffect(() => {
+    if (data.length === 0) dispatch(fetchCurrencies());
+  }, [data.length, dispatch]);/* eslint-enable */
+  return (
+    <div className={styles.container}>
+      <h3>Search Crypto Currencies</h3>
+      <Search />
+      <div className={styles.list_container}>
+        {data.length > 0 ? (
+          data.map((item) => (
+            <NavLink
+              to={`/coin/${item.id}`}
+              key={item.id}
+              onClick={() => set('coins')}
+            >
+              <Coin
+                id={item.id}
+                marketRank={item.market_cap_rank}
+                image={item.image}
+                coinSymbol={item.symbol}
+                currentPrice={item.current_price}
+                imageUrl={item.image}
+                coinName={item.name}
+              />
+            </NavLink>
+          ))
+        ) : (
+          <div>
+            <h1>Loading...</h1>
+            {' '}
+          </div>
+        )}
+      </div>
+    </div>
+  );
+};
+
+export default Home;
```

Comment:
- [ ] Kindly implement the alternating color patterns as we have in the design. As you can see the darker and lighter colors are alternating.

  Your Implementation | Behance Design
  :---:|:---:
  ![ufi](https://user-images.githubusercontent.com/36542986/203125747-b765b3e2-e4fb-49f5-9937-b75e796e96c6.jpg) | ![jonathas-orig-home](https://user-images.githubusercontent.com/36542986/203125759-c50584d6-305a-4c79-80e1-a14431f2e685.jpg)